src/scan.l

```diff
@@ -285,9 +285,24 @@ M4QEND      "]""]"
 }
 
 <CODEBLOCK_MATCH_BRACE>{
+    "'}'"      |
+    "'{'"      |
+    [^{}\r\n]  |
+    \"(\\\"|[^"\r\n])*\"  |
+    "//".*     buf_strnappend(&top_buf, yytext, yyleng);
+
+    [/][*]+([^*]|([*][^/]))*[*]+[/] {
```

Comment:
I wonder if this syntax will match `/*` marks quoted in a literal.
```
"example /* " "/*" } "*/"
```
I think you get the idea.

Reply:
Yes - a seemingly tricky one, but assuming it is stand-alone it should by broken into three string literals with `}` before the last one (other white spaces ignored).  I have not tested. Did you?